Configurations/ConfigCommon.xcconfig

```diff
@@ -104,6 +105,7 @@ XCODE_1310 = -Wno-completion-handler
 XCODE_1320 = -Wno-completion-handler
 XCODE_1330 = -Wno-completion-handler
 XCODE_1340 = -Wno-completion-handler
+XCODE_1400 = -Wno-completion-handler
```

Comment:
We could probably do away with this customization now, given our min versions, but that's beyond the scope of this ticket.